From efd387e0f0816d0f1fd156f8ba75f4b56dcf5034 Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Mon, 7 Mar 2022 14:34:11 +0100
Subject: [PATCH] Don't assume libexec is a LIB_INSTALL_DIR subfolder

FHS 3.0 states that executables not intended to be executed by users shall be under /usr/libexec [1]

[1] https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html
---
 src/kdecore/kstandarddirs.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kdecore/kstandarddirs.cpp b/src/kdecore/kstandarddirs.cpp
index 23064a9a..a61f5ae9 100644
--- a/src/kdecore/kstandarddirs.cpp
+++ b/src/kdecore/kstandarddirs.cpp
@@ -304,7 +304,7 @@ static QString relativeInstallPath(const char *type)
             return QFile::decodeName(LIB_INSTALL_DIR "/");
         }
         if (strcmp("libexec", type) == 0) {
-            return QFile::decodeName(LIB_INSTALL_DIR "/libexec/");
+            return QFile::decodeName(LIBEXEC_INSTALL_DIR "/");
         }
         if (strcmp("locale", type) == 0) {
             return QFile::decodeName(LOCALE_INSTALL_DIR "/");
-- 
2.35.1

